mutiny-core/src/gossip.rs

```diff
@@ -640,7 +640,7 @@ mod test {
     use uuid::Uuid;
     use wasm_bindgen_test::{wasm_bindgen_test as test, wasm_bindgen_test_configure};
 
-    use crate::test::*;
+    use crate::test_utils::*;
```

Comment:
I think the tests at the bottom of this file need `crate_test_utils` too. 

```
    #[test]
    // hack to disable this test
    #[cfg(feature = "ignored_tests")]
    async fn test_gossip() {
        crate::test::log!("test RGS sync");
```

Reply:
made it so `ignored_tests` depends on `test-utils` so it should be safe

Reply:
right but shouldn't it be:

```
 crate::test_utils::log!("test RGS sync");
```

Reply:
oh i see, good catch, fixed